app/services/jobs/return-logs/fetch-return-logs.service.js

```diff
@@ -115,7 +147,17 @@ async function _fetchReturnRequirements (isSummer, licenceReference) {
       builder.select(['id', 'naldRegionId'])
     })
     .withGraphFetched('returnRequirementPoints')
-    .withGraphFetched('returnRequirementPurposes')
+    .modifyGraph('returnRequirementPoints', (builder) => {
+      builder.select(['description',
+        'description',
+        'ngr1',
+        'ngr2',
+        'ngr3',
+        'ngr4'])
+    })
+    .withGraphFetched('returnRequirementPurposes.primaryPurpose')
+    .withGraphFetched('returnRequirementPurposes.secondaryPurpose')
+    .withGraphFetched('returnRequirementPurposes.purpose')
```

Comment:
If you only need the legacy IDs and descriptions off these, then they should have `modifyGraph()` functions which specify this in the select.

Without it, Objection.js will do `SELECT *` pulling everything off the record, and for which I have a strict 🚫 rule.